compiler/optimizer/OrderBlocks.cpp

```diff
@@ -164,8 +164,7 @@ bool TR_OrderBlocks::safeToMoveAway(TR::Block *block, TR::Block *prevBlock)
    if (_changeBlockOrderBasedOnHWProfile)
       return true;
 #if 1
-    if (hasOtherHotAsmFlowPredeccessors(block, prevBlock) ||
```

Comment:
Does it make sense to have a routine named hasOtherHotAsmFlowPredeccessors anymore ? Is it related to the flag(s) that you are removing in some way ?